configurations/project.conf

```diff
@@ -1,7 +1,16 @@
+###############################################################################
+##
+##                  DPS Project Global Environment Variables
+##                  Author: @akroon3r
+##
+###############################################################################
+
 PROJECT_PREFIX="pjztlm"
 PROJECT_NAME="jag-dps"
 PROJECT_OWNER="bcgov"
 GIT_REPO="github.com/${PROJECT_OWNER}/${PROJECT_NAME}.git"
 REPO_REF="master"
 GIT_URL="https://${GIT_REPO}"
-GIT_EMAIL='paul.1.martin@gov.bc.ca'
+GIT_EMAIL="paul.1.martin@gov.bc.ca"
```

Comment:
we should not have personal emails in config

Reply:
aight.

Reply:
removed